### ISSUE TITLE
Cannot read property 'isDirectory' of null

### ISSUE THREAD

**Issue by aramk** [jhermsmeier/gulp-rm#2]
Running this on a hidden directory called ".tmp" gave this error:
```
/Users/.../node_modules/gulp-rm/index.js:67
    if( file.stat.isDirectory() ) {
                 ^

TypeError: Cannot read property 'isDirectory' of null
```
Not sure if we need a null check or if it failed to read.

**Comment by jhermsmeier** [comment]
Thanks for reporting this! Looks like `glob-stream` doesn't populate the `stat` property anymore.
Fixed and published as `gulp-rm@1.0.1`.

**Comment by aramk** [comment]
👍 nice
